synapseclient/api/entity_services.py

```diff
@@ -215,6 +217,80 @@ async def delete_entity_generated_by(
     )
 
 
+async def delete_entity(
+    entity_id: str,
+    version_number: int = None,
```

Comment:
When testing this if you attempt to delete a version of an entity that doesn't exist, nothing happens, but there isn't a warning or error. Is that expected?

Reply:
This is consistent behavior with the current functionality:
```
import asyncio
from synapseclient import Synapse
from synapseclient.api import delete_entity

syn = Synapse()
syn.login()


async def main():
    res = await delete_entity(entity_id="syn53604544", version_number=99)
    print(res)

    res2 = syn.delete(obj="syn53604544", version=99)
    print(res2)

asyncio.run(main())
```